README.md

```diff
@@ -7,7 +7,10 @@ Current backends:
 - Solid
 
 ## Dependencies
-- Solid (http://solid.kde.org) + deps: libupnp, flex/bison, udev
+- Solid (https://api.kde.org/frameworks/solid/html/index.html) 
+  - build deps: flex/bison, udev, extra-cmake-modules
+  - manual build: usually just a matter of `sudo apt build-dep solid` for Ubuntu or `sudo dnf builddep kf5-solid` for Fedora
```

Comment:
`sudo apt build-dep solid` doesn't seem to work unless I have populated the sources.list with the location of the solid source code.

```
➜  poky git:(pyro) sudo apt build-dep libkf5solid
Reading package lists... Done
E: You must put some 'source' URIs in your sources.list
```